LibraBFT/Impl/Consensus/ConsensusTypes/SyncInfo/Properties.agda

```diff
@@ -0,0 +1,53 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
+
+   Copyright (c) 2021, Oracle and/or its affiliates.
+   Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
+-}
+
+open import LibraBFT.Base.Types
+open import LibraBFT.Concrete.System
+open import LibraBFT.Concrete.System.Parameters
+open import LibraBFT.Impl.Properties.Util
+open import LibraBFT.Impl.Types.BlockInfo as BI
+open import LibraBFT.ImplShared.Consensus.Types
+open import LibraBFT.ImplShared.Consensus.Types.EpochDep
+open import LibraBFT.ImplShared.Interface.Output
+open import LibraBFT.ImplShared.Util.Util
+open import LibraBFT.Prelude
+open import LibraBFT.Yasm.System ℓ-RoundManager ℓ-VSFP ConcSysParms
+open import Optics.All
+
+open RoundManagerInvariants
+open RoundManagerTransProps
+
+module LibraBFT.Impl.Consensus.ConsensusTypes.SyncInfo.Properties where
+
+module verifyMSpec (self : SyncInfo) (validator : ValidatorVerifier) where
+
+  record Requirements (pre : RoundManager) : Set where
```

Comment:
This would be a very awkward requirement, since it would have to be bubbled up to all callers of `verifyMSpec` -- and we handle this already with `RoundManagerInv`.

If there are some properties of `SIVerifyProps` that require this in order to be proved, I think it's better to make the final guarantee be a conditional (see below)